### PR TITLE
Add a general front-end to the main page. 

### PR DESCRIPTION
**Pull request by juanuribe28**
The front-end uses mustache to render the specific menu sections. The content of the templates is just there to show this behavior, they don't contain any other meaningful content.

### CODE REVIEW

**Comment by juanuribe28** [comment]
@googlebot I signed it!

File: src/main/webapp/scripts/toggle.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * Returns a promise.all of all the render html sections.
+ */
+async function loadHtmlSections(templatesUrls, objsUrls, sectionsNames) {
+  const htmlTemplatesPromises = loadUrls(templatesUrls, loadTemplate);
+  const objsPromises = loadUrls(objsUrls, loadObject);
+  const values =  await Promise.all([htmlTemplatesPromises, objsPromises]);
+  const templates = values[0];
+  const objs = values[1];
+  let htmlSections = new Object();
+  for (let i = 0; i < sectionsNames.length; i++){
```
Comment (dranilew) [inline src/main/webapp/scripts/toggle.js]: [Nit] Add space before the opening bracket

**Review by dranilew — CHANGES_REQUESTED** [review]
Small nit, everything else looks good to me.

**Review by dranilew — APPROVED** [review]

File: src/main/webapp/scripts/toggle.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const templatesUrls = [keywordsTemplateUrl, newsTemplateUrl, actionsTemplateUrl];
+
+const keywordsObjUrl = '/json/keywords.json';
+const newsObjUrl =  '/json/news.json';
+const actionsObjUrl = '/json/actions.json';
+const objectsUrls = [keywordsObjUrl, newsObjUrl, actionsObjUrl];
+
+const sectionsNames = ['keywords', 'news', 'actions'];
+
+const htmlSectionsPromises = loadHtmlSections(templatesUrls, objectsUrls, sectionsNames);
```
Comment (daperlmut) [inline src/main/webapp/scripts/toggle.js]: For module level constants use CONST_CASE instead of lowerCamelCase

**Review by daperlmut — CHANGES_REQUESTED** [review]

**Review by daperlmut — APPROVED** [review]

**Pull request merged**